recipes/asdf-standard/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "asdf-standard" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_standard-{{ version }}.tar.gz
+  sha256: 60feb42edcb0895a9b78f2e5057e6820c5b11083760983e70a29d856d9f0af84
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm
+  run:
+    - asdf >=2.8.0
+    - importlib_resources >=3  # [py<39]
+    - python
+
+test:
+  imports:
+    - asdf_standard
+  commands:
+    - pip check
```

Comment:
since the dependencies are a little broken right now (until everything is on conda-forge), pip check fails here
you could try running it without this test @WilliamJamieson